Resources/Prototypes/_NF/InventoryTemplates/cat_inventory_template.yml

```diff
@@ -0,0 +1,69 @@
+- type: inventoryTemplate
+  id: nfcat
+  slots:
+  - name: head
+    slotTexture: head
+    slotFlags: HEAD
+    slotGroup: MainHotbar
+    uiWindowPos: 0,0
+    strippingWindowPos: 0,0
+    displayName: Head
+    whitelist:
+      components:
+        - CatWearable
```

Comment:
this is only used for the mail hat so why not remove the component and just have a tag for this as only 1 thing uses it

Reply:
Still hate using tags, but can be.

Reply:
I plan to port more hats in the future

Reply:
ok real